nilearn/glm/tests/test_glm_reporter.py

```diff
@@ -558,28 +557,15 @@ def test_carousel_several_runs(
     assert str(report).count('id="carousel-obj-') == len(shapes)
 
 
-@pytest.mark.slow
-@pytest.mark.parametrize("threshold", [3.09, 2.9, DEFAULT_Z_THRESHOLD])
-@pytest.mark.parametrize("height_control", [None, "bonferroni", "fdr", "fpr"])
-def test_report_threshold_deprecation_warning(
-    flm, contrasts, threshold, height_control
-):
-    """Check a single warning thrown when threshold==old threshold.
+def test_report_make_glm_deprecation_warning(flm, contrasts):
+    """Test deprecation warning for nilearn.reporting.make_glm_report.
 
     # TODO (nilearn >= 0.15)
     # remove
     """
-    with warnings.catch_warnings(record=True) as warning_list:
-        flm.generate_report(
+    with pytest.warns(DeprecationWarning):
+        make_glm_report(
+            flm,
             contrasts=contrasts,
-            threshold=threshold,
-            height_control=height_control,
+            threshold=3.9,
```

Comment:
Why this threshold ? 3.09 would be a more standard choice (corresponding to  p< 0.001)